.github/workflows/test.yml

```diff
@@ -10,6 +10,14 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
     steps:
+      - name: Get GH Access Token
+        uses: Bandwidth/vault-provider-action@v1
+        with:
+          identity-source: repo
+          durable-team-name: BAND SWI
+          artifactory-access-token-type: readers
```

Comment:
```suggestion
```

Reply:
We shouldn't need to include this as we're not doing anything with Artifactory access.